Makefile

```diff
@@ -193,7 +203,10 @@ dockerbuild-test:
 .PHONY: dockerbuild-test
 
 
-dockerpush: dockerbuild-go
+dockerpush: dockerbuild-go dockerpush-only
```

Comment:
For the record, I find the naming scheme of these make targets very confusing. It probably doesn't matter too much since we don't use them outside of make.

Reply:
Yeah, I agree but it requires a bigger refactor than I'm willing to make for this small change...